src/main/java/net/glowstone/block/blocktype/BlockType.java

```diff
@@ -303,10 +306,16 @@ public final void rightClickBlock(GlowPlayer player, GlowBlock against, BlockFac
             return;
         }
 
-        // TODO: check bounding box
         if (getMaterial().isSolid()) {
-            for (Entity entity : against.getChunk().getEntities()) {
-                if (entity instanceof LivingEntity && entity.getLocation().getBlockX() == target.getX() && entity.getLocation().getBlockY() == target.getY() && entity.getLocation().getBlockZ() == target.getZ()) {
+            BlockBoundingBox box = new BlockBoundingBox(target);
+            BoundingBox searchBox = BoundingBox.copyOf(box);
+            Vector vec = new Vector(32, 32, 32);
```

Comment:
Wait so here are you expanding the block by 32? Why not just keep it the same size?